ui/app/AppLayouts/Profile/panels/SupportedTokenListsPanel.qml

```diff
@@ -49,23 +47,6 @@ StatusListView {
         ]
     }
 
-    footer: Item {
-        width: parent.width
-        height: root.count > 0 ? shapeRect.implicitHeight + root.spacing : shapeRect.implicitHeight
-
-        ShapeRectangle {
-            id: shapeRect
-
-            anchors.bottom: parent.bottom
-            anchors.horizontalCenter: parent.horizontalCenter
-            width: parent.width - 2 // The rectangular path is rendered outside
-            height: ProfileUtils.defaultDelegateHeight
-
-            icon: "add"
-            text: qsTr("Add a Token List (coming soon)")
```

Comment:
Decided to remove this part... we've had it "coming soon" for 2 years already :laughing: